### ISSUE TITLE
Panning doesn't work in the newest version

### ISSUE THREAD

**Issue by L12C** [lucasnlm/antimine-android#460]
**Describe the bug**
As of the newest version, I cannot move the field around, which makes it impossible to play large field sizes

**To Reproduce**
1. Start a new game
2. Try moving the field around

**What control style do you use?**
Single tap to flag, long tap to open

**Expected behavior**
The board should be able to pan around.

**Screenshots**

https://github.com/lucasnlm/antimine-android/assets/56705486/f34dec78-a2f9-4b77-8923-5c4f5c1dbec9

**Android Version:**
13

**App Version**
17.4.0 F

**Comment by Playit3110** [comment]
Same issue here

Antimine: 17.4.0 F
Android 13
Samsung S20

**Comment by lucasnlm** [comment]
This happens due to a bug in the latest (1.12.0) version of GDX. I will revert to the previous version to fix this problem. I hope to release the fix this week.

**Comment by lucasnlm** [comment]
Hi again!
Folks, I just released the `17.5.0` version. I would really appreciate if you test the apk provided here: https://github.com/lucasnlm/antimine-android/releases/tag/17.5.0

It will take a few days untill F-droid updates with this version, so this APK is the fastest way to let me know if it's all right or not.

**Comment by Abi45** [comment]
Hi Lucas, I tried the apk and it fixes the panning problem. I did notice, that all the games now start with the "open" button selected instead of the "flag" button. This has caused me to start a lot of games by exploding mines out of habit!
I don't know if this change was deliberate, but if so, it would be nice in a future version to have a way to select which one should be the default when starting a game.
